modules/administration-guide/partials/assembly_uninstalling-the-che-instance-and-the-che-operator.adoc

```diff
@@ -1,7 +1,52 @@
 [id="uninstalling-the-che-instance-and-the-che-operator_{context}"]
 = Uninstalling the {prod-short} instance and the {prod} Operator
 
-After uninstalling the {devworkspace} Operator, uninstall the {prod-short} instance and the {prod} Operator by using either the OpenShift web console or the OpenShift CLI.
+After uninstalling the {devworkspace} Operator, uninstall the {prod-short} instance and the {prod} Operator.
```

Comment:
should this be `{prod-short}` ?

Reply:
AFAIK, {prod} Operator is fine. That would call for another issue.